.github/workflows/ci.yml

```diff
@@ -13,76 +13,103 @@ on:
     paths-ignore:
       - '**.md'
 
-# Cancels all previous workflow runs for the same branch that have not yet completed.
```

Comment:
Leave the concurrency in 🙏🏼

Reply:
@dingo-d, sure, but I don't know how to fix these CI issues. I can remove these changes entirely, this was just testing to see how to fix CI.

Unfortunately this is still stuck on Composer failing to authenticate against GitHub.